ol/cli/src/node/query.rs

```diff
@@ -667,11 +826,23 @@ fn test_is_community_wallet_should_return_false_if_missing_unfreeze_votes() {
 fn test_is_community_wallet_should_return_false_if_missing_consecutive_rejections() {
     let value = vec![
         (
-            Identifier::new("is_frozen").unwrap(),
+            match Identifier::new("is_frozen") {
```

Comment:
How about using one line expect here instead of match? I believe, it should do the exact same as the following 5 lines.
`Identifier::new("is_frozen").expect("")`
https://doc.rust-lang.org/std/result/enum.Result.html#method.expect

You can also use in other places:
`Identifier::new("is_frozen").expect("my error msg")`

Reply:
Seems to me, there are a few places that you cannot use expect - for all changes in this PR. You should be able to use expect instead of match for 90% of the `match`es in these code changes.

Reply:
Fixed a lot of these. Thanks for the explanation.